src/gettsim/tests_germany/policy_cases/wohngeld/2006-01-01/hh_id_15.yaml

```diff
@@ -314,6 +323,15 @@ inputs:
             - 0.0
             - 0.0
       rente:
+        jahr_renteneintritt:
```

Comment:
Just a random occurrence: Why does this pop up as a result of the changes in this PR? I don't quite see it yet.